### PR TITLE
fix some docs

### PR DESCRIPTION
**Pull request by jonay2000**
This fixes some documentation issues we encountered while using the library.

### CODE REVIEW

File: src/font.rs [hunk 1]
```diff
@@ -67,8 +67,10 @@ impl<'a> Font<'a> {
     ///  - `text`: text to render.
     ///  - `row`: x (Left/Right align) or y (Up/Down align) offset where to stop
     ///    rendering.
-    ///  - `wh`: the size of each character in X & Y dimensions.
     ///  - `text_align`: how the text is aligned
+    /// 
+    ///  Returns an iterator which generates the path from characters (see [`TextPathIterator`])
+    ///  and a number indicating what was leftover
```
Comment (jonay2000) [inline src/font.rs:73]: The leftover number should be documented better, but I'm not entirely sure what it's measured in. Maybe provide an example of how to use it?

File: src/font.rs [hunk 2]
```diff
@@ -67,8 +67,10 @@ impl<'a> Font<'a> {
     ///  - `text`: text to render.
     ///  - `row`: x (Left/Right align) or y (Up/Down align) offset where to stop
     ///    rendering.
-    ///  - `wh`: the size of each character in X & Y dimensions.
```
Comment (jonay2000) [inline src/font.rs:70]: This parameter doesn't seem to be there. Maybe it's leftover docs from a previous version.

Comment (AldaronLau) [reply src/font.rs:70]: Yeah, this got replaced by using Transforms from the footile crate, so it shouldn't be there anymore.

File: src/font.rs (hunk 1 above)
Comment (AldaronLau) [reply src/font.rs:73]: It's the number of characters not rendered.  I can add an example on how to use it.

**Comment by AldaronLau** [comment]
Looks good, merging now

**Pull request merged**